elastic/datadog_checks/elastic/__init__.py

```diff
@@ -2,6 +2,6 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 from .__about__ import __version__
-from .elastic import ESCheck
+from .elastic import ESCheck, AuthenticationError
 
-__all__ = ['__version__', 'ESCheck']
+__all__ = ['__version__', 'ESCheck', 'AuthenticationError']
```

Comment:
You do not have to modify this file in fact and you could `from datadog_checks.elastic.elastic import AuthenticationError` instead where you need it

Reply:
Any specific reason not to modify this file and import it directly?

Reply:
because it's not needed to add extra code to import it somewhere else